docs/adr/0022-grant-output-checkers-view-released-file.md

```diff
@@ -0,0 +1,33 @@
+# 22. Grant Output Checkers Permission to View Released Files
+
+Date: 2024-08-14
+
+## Status
+
+Accepted
+
+## Context
+
+Output Checkers need to view previously-released files when asssessing a file requested for release in order to understand:
+* [Secondary Disclosure][1] risks from combination of the requested file(s) with previous releases from this project
+* The context of decisions previously made when output checking previous file versions (e.g. if previously rejected, for what reason and what may have changed since)
+
+This need has been previously satisfied by assigning output checkers the global `ProjectCollaborator` role (per the [team manual][1]).
+This further confuses the notion of what a "collaborator" role is for a project, and what/why permissions may be associated with this role.
+
+
+
+## Decision
```

Comment:
nit:

```suggestion

## Decision
```

There's similar additional whitespace below, before the footnotes, but I've also another edit suggestion there too, so I'll leave it for manual editing.

(It might be better if we ran a Markdown linter, but we currently don't.)